packages/packages/libs/editor-controls/src/controls/transition-control/transition-repeater-control.tsx

```diff
@@ -48,18 +51,51 @@ function getChildControlConfig( recentlyUsedList: string[] ) {
 	};
 }
 
-export const TransitionRepeaterControl = createControl( ( props: { recentlyUsedList: string[] } ) => {
-	return (
-		<RepeatableControl
-			label={ __( 'Transitions', 'elementor' ) }
-			repeaterLabel={ __( 'Transitions', 'elementor' ) }
-			patternLabel="${value.selection.value.key.value}: ${value.size.value.size}${value.size.value.unit}"
-			placeholder={ __( 'Empty Transition', 'elementor' ) }
-			showDuplicate={ false }
-			showToggle={ true }
-			initialValues={ initialTransitionValue }
-			childControlConfig={ getChildControlConfig( props.recentlyUsedList ) }
-			propKey="transition"
-		/>
-	);
-} );
+const disableAddItemTooltipContent = (
+	<Alert
+		sx={ {
+			width: 280,
+			gap: 0.5,
+		} }
+		color="secondary"
+		icon={ <InfoCircleFilledIcon /> }
+	>
+		<AlertTitle>{ __( 'Transitions', 'elementor' ) }</AlertTitle>
+		<Box component="span">
+			<Typography variant="body2">
+				{ __( "Switch to 'Normal' state to add a transition.", 'elementor' ) }
+			</Typography>
+		</Box>
+	</Alert>
+);
+
+export const TransitionRepeaterControl = createControl(
+	( {
+		recentlyUsedList,
+		currentStyleState,
+	}: {
+		recentlyUsedList: string[];
+		currentStyleState: StyleDefinitionState;
+	} ) => {
+		const currentStyleIsNormal = currentStyleState === null;
+		return (
+			<RepeatableControl
```

Comment:
Don't we want to replace the repeatable control with the new `unstableRepeatableControl`
WDYT?